Resources/Textures/Tiles/attributions.yml

```diff
@@ -109,4 +109,9 @@
 - files: ["chromite.png"]
   license: "CC-BY-NC-SA-3.0"
   copyright: "taken at commit 0587dd16e28108bdf0b0a28e2caae4319845e861, and recolored by TheShuEd"
-  source: "https://github.com/Mojave-Sun/mojave-sun-13"
\ No newline at end of file
+  source: "https://github.com/Mojave-Sun/mojave-sun-13"
+
+- files: ["mining_floor.png", "mining_floor_dark.png", "mining_floor_light.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "taken at https://github.com/ParadiseSS13/Paradise/commit/43889a89d5a9378fd120d627f74613edb1841a66"
+  source: "https://github.com/ParadiseSS13/Paradise"
```

Comment:
These don't looke like they're in goonstation folders hence wouldn't be NC assets.

Reply:
They were taken from Paradise Station.

Reply:
Fixed